docs/usage.md

```diff
@@ -1,40 +1,97 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+| -------------------- |
 
 # Usage
 
-The Record Distribution widget provides the ability to visualise records or items based on a specified grouping. For example, viewing the distribution of assets by purdue levels, or the distribution of alerts by type, severity, etc.
+The **Record Distribution** widget helps visualise records or items based on a specified grouping criterion. For example, viewing the distribution of assets by purdue levels, or the distribution of alerts by type, severity, or other criteria.
 
-- Ability to group records using a picklist, say "Levels" in the "Asset" module. The benefit of having a grouped view is that you can view all of the records together based on the grouping. In the case of our example, it would be view assets at each level. Additionally, it also provides a visual depiction of each record node in the group.
-- Ability to add this view on a dashboard, report, or within the detail view of a module's record. 
+You can group records using a picklist, say *Levels* in the **Assets** module. With such a criterion you can view all of the assets together and at each level. Additionally, it also provides a visual depiction of each record node in the group.
 
-An example of using the Record Distribution widget would be to display the assets grouped by their levels.
+You can add this view to a dashboard, report, or within the detailed view of a module's record.
```

Comment:
This info can be added for all widgets going forward in overview section like we mention about version info. The title for the same could be compatibility